Django Internal Tests: 500 Error